src/main/java/com/thevoxelbox/voxelsniper/brush/type/performer/SplineBrush.java

```diff
@@ -73,6 +74,15 @@ public void handleCommand(String[] parameters, Snipe snipe) {
         }
     }
 
+    @Override
```

Comment:
Putting this here, but using `ren` would produce an AIOOBE if used before setting control- and endpoints.